static/css/style.css

```diff
@@ -51,13 +65,13 @@ footer {
 /* Not required for template or sticky footer method. */
 
 .login {
-	padding: 10px 0px 5px 990px; 
+float:left;
+
 }
 h1 {
 	width: 350px;
-	text-align: center;
 	position: relative; 
-	margin:-100px 0 0 -150px;
+	margin: 45px 0px 105px -150px;
```

Comment:
If the fixed navbar overlapping the content (usually the h1 title), you can simply add to the container class a margin-top: 60px. Yo don't need to modify the h1,h2,h3 elements styling or to define a new one with ID as you did.
Please remove the h1, h2, h3 modifications from the style.css.